crates/edr_napi/src/provider.rs

```diff
@@ -245,6 +247,31 @@ impl Provider {
     }
 }
 
+/// Tracing config for Solidity stack trace generation.
+#[napi(object)]
+pub struct TracingConfigWithBuffers {
+    /// Build information to use for decoding contracts.
+    pub build_infos: Option<Vec<Uint8Array>>,
```

Comment:
Wrapping in an `Option` is a change from the design doc, but it maintains backwards compatibility as the `TracingConfig` in Hardhat has the `buildInfo` as an optional parameter.